plugins/catalog-react/src/filters.ts

```diff
@@ -159,3 +163,28 @@ export class UserListFilter implements EntityFilter {
     return this.value;
   }
 }
+
+/**
+ * Filters entities based if it is an orphan or not.
+ * @public
+ */
+export class EntityOrphanFilter implements EntityFilter {
+  constructor(readonly values: string[]) {}
+  filterEntity(entity: Entity): boolean {
+    const orphan = entity.metadata.annotations?.['backstage.io/orphan'];
+    return orphan !== undefined && this.values.includes(orphan);
+  }
+}
+
+/**
+ * Filters entities based on if it has errors or not.
+ * @public
+ */
+export class EntityErrorFilter implements EntityFilter {
+  constructor(readonly values: string[]) {}
+  filterEntity(entity: Entity): boolean {
+    const error =
```

Comment:
Probably can be a future enhancement but it would be cool to emulate the [`ErrorsPanel`](https://github.com/backstage/backstage/blob/d9f86c2d81d85975d89729df905c2f4c26736060/plugins/catalog/src/components/EntityProcessingErrorsPanel/EntityProcessingErrorsPanel.tsx#L55-L65) and indicate if any part of the ancestry has an error (though there's probably performance concerns grabbing that info)

Reply:
Oh, that would be pretty cool! Feel like that could come in a later update though